packages/renderer/src/App.svelte

```diff
@@ -93,7 +93,7 @@ window.events?.receive('display-troubleshooting', () => {
         <Route path="/" breadcrumb="Dashboard Page">
           <DashboardPage />
         </Route>
-        <Route path="/containers" breadcrumb="Containers" navigationHint="root">
+        <Route path="/containerslist/*" breadcrumb="Containers" navigationHint="root">
```

Comment:
could you explain why we need to have a dedicated url for containerslist rather than having it part of /containers/ ? or `/containers/list` for example

Reply:
There is another route `/containers/:id`, and it seems it will have priority over `/containers/list`, or `/containers/anything`.

I tried to move the `/containers/*` before `/containers/:id` in the list of Reoutes, but the priority didn't change.

Is there a way to change this priority?

Reply:
based on https://github.com/AlexxNB/tinro#show-first-matched-nested-route-only

looks like `/containers/list`. would work but maybe you want to have `/containers/list/*` and `/containers/details/...`

might be better if all routes for containers are in a single route prefix